.github/workflows/github-workflow-badger.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: github-merit-badger
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  call-action:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: aws-github-ops/github-merit-badger@v0.0.98
```

Comment:
does this need to be allowed for the org?

i know in the past if the github action was not explicitly defined at the org wide settings that it will fail (at least for write permissions)